presentation/build.gradle

```diff
@@ -44,9 +44,11 @@ dependencies {
     implementation SupportDependencies.firebaseAnalytics
 
     implementation LibsDependencies.glide
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
Why did you add this two dependencies? They are already added.

Reply:
DoneI deleted them, they were not necessary